src/test/controllerengine_test.cpp

```diff
@@ -36,6 +36,16 @@ class ControllerEngineTest : public MixxxTest {
                                         QScriptValueList());
     }
 
+    void processEvents() {
+        // QCoreApplication::processEvents() only processes events that were
+        // queued when the method was called. Hence, events that were emitted
+        // by a slot that was triggered by processing are *not* processed and
```

Comment:
Do we use `*not*` or `NOT` to highlight text in comments? Asking for future reference.

Reply:
No idea. Enclosing a string in `*` denotes an emphasis in markdown, and when generating doxygen documentation, the string will by displayed in italics.